test/suites/idmap.sh

```diff
@@ -219,4 +221,37 @@ EOF
   [ "$(stat -c '%u:%g' "/proc/${PID}/root/d")" = "$((UID_BASE+29)):$((GID_BASE+29))" ]
 
   lxc delete idmap --force
+
+  # Check that we have a big enough range for this test
+  if [ "${SUBID_FILES_EXISTED}" -eq 0 ]; then
+    UIDs=1000000000
+    GIDs=1000000000
+    UID_BASE=0
+    GID_BASE=0
+  else
+    UIDs=1000000000
+    GIDs=1000000000
+    UID_BASE=0
+    GID_BASE=0
+
+    mv /etc/subuid /etc/subuid_backup
+    mv /etc/subgid /etc/subgid_backup
+
+    echo "lxd:"$UID_BASE":$UIDs" > /etc/subuid
+    echo "lxd:"$UID_BASE":$GIDs" > /etc/subgid
+    echo "root:"$GID_BASE":$UIDs" > /etc/subuid
+    echo "root:"$GID_BASE":$GIDs" > /etc/subgid
```

Comment:
We absolutely can't do that kind of stuff in the testsuite, a failure in the middle of this code would break the system and when running multiple testsuites in parallel, you'll end up with undefined results.

We can't have the testsuite modify system files like that.

Reply:
ok